app/src/mapPanels/rightControlPanel/components/MenuLink.jsx

```diff
@@ -2,13 +2,17 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import MenuLinkStyles from 'styles/components/menu-link.scss';
 import ControlPanelStyles from 'styles/components/control_panel.scss';
+import classnames from 'classnames';
 
-function MenuLink({ title, icon, onClick }) {
+function MenuLink({ title, icon, badge, onClick }) {
   return (
     <div className={MenuLinkStyles.menuLink} onClick={onClick}>
       <div className={MenuLinkStyles.header} >
         <h2 className={MenuLinkStyles.title} >{title}</h2>
-        <div className={ControlPanelStyles.icon} >{icon}</div>
+        <div className={classnames(MenuLinkStyles.rightIcons, { [MenuLinkStyles.hasBadge]: badge > 0 })} >
```

Comment:
Let's try to keep the css conventions we have in the project: modifier classes should begin with a _, see https://github.com/Vizzuality/GlobalFishingWatch/pull/703

Reply:
In this case is not a modifier, is the container to the icon and badge on the right. Maybe it can use a better name